internal/storage/driver/memory/binding.go

```diff
@@ -40,6 +40,18 @@ func (s *Binding) Insert(binding *internal.Binding) error {
 	return nil
 }
 
+func (s *Binding) Update(binding *internal.Binding) error {
+	s.mu.Lock()
+	defer s.mu.Unlock()
+
+	if _, found := s.data[binding.ID]; !found {
```

Comment:
Should we put a comment here, that this behaviour is different than this of postgres?

Reply:
Corrected in https://github.com/kyma-project/kyma-environment-broker/pull/1366/commits/8700a18713ab50158aab98877d675ba8b8da66aa